### ISSUE TITLE
MTB:scale of 0- missing from the path details (Vespucci) 

### ISSUE THREAD

**Issue by TommiContursi** [simonpoole/beautified-JOSM-preset#461]
Hello!

I have observed that it is currently not possible to add MTB:scale of 0-. I assume the use of negative zero is relatively new, but it is widely used here in Finland with the Trailmap app. In this context, the zero indicates easy forest mtb trails, and the negative zero distinguishes these from forest jogging paths, roads, and other very easy trails.

https://wiki.trailmap.fi/fi/kartoitus/tagit

**Comment by simonpoole** [comment]
This is a bit of a quagmire as adding all the intermediate values to the filed would triple the number of values with not a lot of benefit.  And I'm somewhat at a loss as to what could be easier than 0 to start with (no corners? :-)).

I might simply just add the most used intermediate values: 0- and 0+

**Comment by TommiContursi** [comment]
Yup, it might be unnecessary to include any of these, which could confuse some users. I just checked, and it seems like 0- is mainly used here in Finland. https://taginfo.openstreetmap.org/tags/mtb%3Ascale=0-#map

Here are the most frequently used values. https://taginfo.openstreetmap.org/keys/mtb%3Ascale#values
